release/models/bgp/openconfig-bgp.yang

```diff
@@ -68,7 +68,14 @@ module openconfig-bgp {
     whereas leaf not present inherits its value from the leaf present
     at the next higher level in the hierarchy.";
 
-  oc-ext:openconfig-version "9.8.0";
+  oc-ext:openconfig-version "9.9.0";
```

Comment:
This model doesn't appear to need any edits at all - no version increment is needed